lib/client/Client.js

```diff
@@ -110,33 +156,34 @@ export default class Client {
    * @param deviceName {String} A unique device name
    * @param continuationCallback {Function} A function that takes an {@link AuthContinuation} object as argument, and should return a promise, that will resolve with the continuation token.
    *
-   * @return {Client} This Client instance.
+   * @return {Promise} A {@link Promise} that will eventually be resovled with a {@link AuthAccess} object
    */
-
   authExternal(username, deviceName, continuationCallback) {
```

Comment:
I'm not sure if I understand correctly. Isn't the `continuationCallback` meant for the client app to notify the library when the "external" authentication process is complete and the JMAP authentication can continue? From the spec:

> The client MUST also offer a continue button (or similar) for the user
> to indicate to the client when   they have completed the out-of-band
> authentication.

I suppose a click on that button will then resolve the promise returned by the `continuationCallback`. How would one use the `authExternal` without such a callback?